ci-operator/step-registry/openshift/upgrade/azure/loki/openshift-upgrade-azure-loki-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-upgrade-azure-loki
+  steps:
+    pre:
+    - ref: ipi-install-monitoringpvc
+    - ref: ipi-install-loki
+    - chain: ipi-azure-pre-stableinitial
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: gather-loki
+    - chain: ipi-azure-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
```

Comment:
as latest is older then this PR should it be from latest to initial?

Reply:
Names are a bit confusing, but `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE` is used in `ipi-install-install` (so its initial) and `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` used in `openshift-e2e-test` doing `openshift-tests run-upgrade --to-image=${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}`